cli/src/config.rs

```diff
@@ -53,13 +53,43 @@ pub struct Configuration {
     /// Configuration for `WorldStateView`.
     #[config(inner)]
     pub wsv: WorldStateViewConfiguration,
-    #[cfg(feature = "telemetry")]
-    /// Configuration for telemetry
-    #[config(inner)]
-    pub telemetry: iroha_telemetry::Configuration,
     /// Network configuration
     #[config(inner)]
     pub network: NetworkConfiguration,
+    /// Configuration for telemetry
+    #[config(inner)]
+    #[cfg(feature = "telemetry")]
+    pub telemetry: iroha_telemetry::Configuration,
+}
+
+impl Default for Configuration {
+    #[allow(clippy::expect_used)]
+    fn default() -> Self {
+        let public_key = "ed01201c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b"
```

Comment:
if #2104 gets merged first, the generator needs to be made aware of this change.

Reply:
ok, let's be mindful of this